scss/_accordion.scss

```diff
@@ -58,39 +56,36 @@
 }
 
 .accordion-item {
+
   &:first-of-type {
+
     .accordion-button {
       @include border-top-radius($accordion-border-radius);
     }
   }
 
+  // Only set a border-radius on the last item if the accordion is collapsed
   &:last-of-type {
+    @include border-bottom-radius($accordion-border-radius);
     border-bottom: $accordion-border-width solid $accordion-border-color; // Boosted mod: borders are set on .accordion-header
 
     .accordion-button {
       // Only set a border-radius on the last item if the accordion is collapsed
       &.collapsed {
-        // Boosted mod: borders are set on .accordion-header
         @include border-bottom-radius($accordion-border-radius);
```

Comment:
wrong variable value according to bootstrap need to be `accordion-inner-border-radius`and so lint won't cry for unusued variable